.github/workflows/ci.yml

```
@@ -64,5 +64,20 @@ jobs:
         uses: actions/upload-artifact@5d5d22a31266ced268874388b861e4b58bb5c2f3 # v4.3.1
         with:
           name: cover-${{ github.run_id }}.html
-          path: ${{ github.workspace }}/cmd-cover.html
+          path: ${{ github.workspace }}/cover.html
```

If we want, we can upload both coverage reports. I think this should work:
```suggestion
          path: |
            ${{ github.workspace }}/cover.html
            ${{ github.workspace }}/generator-cover.html
```